### PR TITLE
Feature: Added a description to each `IAction`

### LINKED ISSUE #11786: Feature: Add a description to each `IAction`
### What feature or improvement do you think would benefit Files?

I think a description of what the action does would be helpful when implementing the command palette or shortcut customization features.

### Requirements

- `IAction` must have a property named `Description`
- `Description` property of each action must have a description of what the action does

### Files Version

2.4.54.0

### Windows Version

Windows 11 22H2

### Comments

_No response_

### PR DESCRIPTION
**Pull request by hishitetsu**
**Resolved / Related Issues**
- [X] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #11786 

**Validation**
How did you test these changes?
- [X] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [X] Did you remove any strings from the en-us resource file?
   - [X] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### CODE REVIEW

**Comment by yaira2** [comment]
Once this is finished, we can port it over the to the website repo.

**Comment by hishitetsu** [comment]
Why is there no hotkey assigned to OpenItem? I believe Enter should be assigned.

**Comment by hishitetsu** [comment]
I filled in all the descriptions.

**Comment by QuaintMako** [comment]
@hishitetsu I've added a few commands I forgot in #11481.

**Comment by hishitetsu** [comment]
Don't worry! If the list is missing an update, I will maintain it.
However, it might be a good idea to add a check to the PR template.

**Comment by yaira2** [comment]
Should we leave this open until we finish with the rich command work?

**Comment by hishitetsu** [comment]
That's also possible. I hope to merge this by the time we start implementing the command palette or shortcut customization.

**Review by 0x5bfa — CHANGES_REQUESTED** [review]
Please don't add spaces in table.

**Comment by hishitetsu** [comment]
> Please don't add spaces in table.

You mean '| Category      |' should be '| Category |' for example?

**Comment by onein528** [comment]
Yes. and separator should be `---` each

**Comment by hishitetsu** [comment]
At first I did so, but @yair100 put in the space. I think it is easier to see it as it is now, but why shouldn't it include spaces?

**Comment by 0x5bfa** [comment]
@yair100 This kind indentation will be meaningless even if it is readability by following two major problems.

- When you have to add a new text that has more letters than expected.
- When you have to edit a existing text, you have to adjust spacing.

I also thought this is nice idea before, but it's tiresome, annoying.

**Comment by yaira2** [comment]
@hishitetsu do you want to add the descriptions to the resource file now? I think we have enough to move forward with this now.

**Comment by hishitetsu** [comment]
> @hishitetsu do you want to add the descriptions to the resource file? I think we have enough to move forward with this.

Ok, I will do it within a few days.

**Comment by hishitetsu** [comment]
> Ok, I will do it within a few days.

Done.

File: specs/RichCommand/CommandList.md
Comment (yaira2) [inline specs/RichCommand/CommandList.md:1]: Do we still need this file, or should we move the information over to the website documentation?

Comment (hishitetsu) [reply specs/RichCommand/CommandList.md:1]: I think it depends on whom the website documentation is targeted to, as this document contains information that is not needed by the average user.

Comment (yaira2) [reply specs/RichCommand/CommandList.md:1]: The documentation is targeted towards users, but once this PR is merged, we won't need it for ourselves anymore since it already served its purpose of tracking the different actions.

Comment (hishitetsu) [reply specs/RichCommand/CommandList.md:1]: It may be necessary for the purpose of checking for similar actions already in place if developers want to add actions in the future. However, I leave it to you to decide where to put this.

Comment (yaira2) [reply specs/RichCommand/CommandList.md:1]: We can do both, we'll just have to keep the two in sync.

**Review by yair100 — APPROVED** [review]
LGTM

**Pull request merged**
